db/migrate/20180501072044_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.integer :uid, null: false # this is the identifier provided by GitHub
```

Comment:
This project already has a table `users`, so creating a new table probably isn't what you want here. Instead you might edit the existing table using `add_column`.

Looks like Rails takes the last one if there are multiple migrations with the same name, so this may have worked accidentally.